### ISSUE TITLE
New particle effects fail

### ISSUE THREAD

**Issue by T-oasterO-ven** [mastercoms/tf2cfg#14]
* How you encountered the issue
After using this updated config the new jungle inferno particles started to flood my console with failed 'pre-cache' errors and failed 'creation' errors (invisible flames [including Dragon's Fury balls, Thermal Thruster effects]).  BTW i've **used both vendor/device launch option fixes** (Your fix, and another's [current one]) **both don't work with your config, but work with the old config I used**. Yes it's my laptop, not my desktop. I've also tried the dxlevel 95 command--doesn't work.
    
    * What might be wrong:
Might be a command in this config that specifically messes up the particles (for me at least), or it's missing a command that my other (older) config had; I don't know.

    * Config version
Latest
   
    * Config preset(s)
Maxperformance, igpu 
    
    * Custom values (if any)
none
   
    * DX level
90
    
    * Launch options
-nojoy -console -full -w 1280 -h 720 -novid -dxlevel 90 -nosteamcontroller -noff -nohltv -softparticlesdefaultoff -reuse -primarysound -snoforceformat -swapcores -limitvsconst -force_vendor_id 0x10DE -force_device_id 0x1180
    
    * Operating system (OS)
Linux Ubuntu 16.04 LTS
  
    * Graphics card (GPU)
Intel® HD Graphics 400 (Braswell)
    
    * Processor (CPU)
Intel® Celeron(R) CPU N3060 @ 1.60GHz × 2 
  
    * HDD or SSD
SSD
    
    * RAM size in MB or GB
3.8 GB

    Older config I used with vendor/device launch options (new particles worked) (fused some of your config into it recently; and the new particles still work):
[autoexec(outdated).zip](https://github.com/mastercoms/tf2cfg/files/1424657/autoexec.outdated.zip)

I just want to use your ongoing cfg with no errors.

Thank you.

**Comment by mastercoms** [comment]
Not sure if it is a pasting error but `-force_vendor_id 0x10DE -force_device_i` is not a complete launch option, it should be `-force_vendor_id 0x10DE -force_device_id 0x1180`.

You can try commenting out `mat_dxlevel 95` in the Init section of comfig.cfg, and just use a -dxlevel option, try 90, 95 or 98.

Also try removing `-nod3d9ex`.

On a side note, remove `-noipx`. It doesn't exist in the Source Engine.

**Comment by T-oasterO-ven** [comment]
Wow, that actually worked! Followed **all** your suggestions using `-dxlevel 90`.

I kinda **want to know exactly** what fixed that because beforehand I tried the full: 
`-force_vendor_id 0x10DE -force_device_id 0x1180`; but it did not work... so I asked the tf2 steam forums and one guy gave me the "unfinished cmd" so tried it (did not work) but it worked with the old cfg though. 

Was it the `mat_dxlevel` cmd along with the dx support (max performance) cfg?
Or the `-nod3d9ex` 
Or was it just the "paste error"

I'll have to **do more testing** to find out **exactly** what the issue was caused by, so it will be **easier to identify** in the future. 

I'll report back with it "rooted out"

Thanks
